types/errors/errors.go

```diff
@@ -358,6 +362,17 @@ func WithType(err error, obj interface{}) error {
 	return Wrap(err, fmt.Sprintf("%T", obj))
 }
 
+// IsOf checks if a received error is caused by one of the target errors.
+// It extends the errors.Is functionality to a list of errors.
+func IsOf(received error, targets ...error) bool {
```

Comment:
We could further improve this by declaring its intent as: AnyOf or OfAny. We could also improve the comment and make it more precise by saying that the function returns true if any of the targets satisfy errors.Is for the subject error.

Reply:
But those are two different functions - we would need to create `AnyOf`. This is what you have in mind?
Ref: https://pkg.go.dev/errors#As